SU-SOE/soe_basic
# D8CORE-3528: updating class name to match reality

# READY FOR REVIEW

# Summary
- Updating the block class name for the superfooter to match correctly.

# Review By (Date)
- 2/23

# Urgency
- hig

# Steps to Test

1. pull in this change.
2. pull in https://github.com/SU-SWS/stanford_profile_helper/pull/75
3. Clear caches.
4. Add a super footer with links
5. Verify it has the SOE look and the icons are appearing.

# Affected Projects or Products
- https://github.com/SU-SWS/stanford_profile_helper/pull/75
- Stanford-Profile-Helper

# Associated Issues and/or People
- D8CORE-3528
- D8CORE-3516
- https://github.com/SU-SWS/stanford_profile_helper/pull/75

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
